<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePanel</class>
 <widget class="QWidget" name="HomePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="homeLayout">
   <item>
    <widget class="QLineEdit" name="search_input">
     <property name="placeholderText">
      <string>Search packages...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="package_scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="package_container">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>782</width>
        <height>69</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>